Reformat the HTML in a bunch of my blog posts

realsimonwu opened the pull request:
When I tried to view blogposts I transcribed using the development server I noticed that they weren't actually being rendered as HTML. I reformatted a bunch of the blogposts I transcribed to make it actually render.

realsimonwu committed:
Reformat the HTML in a bunch of my blog posts

When I tried to view blogposts I transcribed using the development server I noticed that they weren't actually being rendered as HTML. I reformatted a bunch of the blogposts I transcribed to make it actually render.

hirundinidae commented on _digest/january-2021/will-ai-drive-healthcare-workers-into-extinction.mdx:
```diff
@@ -6,229 +6,139 @@ summary: Artificial intelligence is already disrupting global economics systems
+					However, radiologists still prescribe appropriate treatment plans and counsel patients; a task AI simply cannot do effectively.</p>
+				<p>Such can also be said about epidemiology and pathology, where AI predictive models are already being used to predict and monitor disease outbreaks and disease progression respectively.</p>
+				<p>Furthermore, though some menial tasks done by pharmacists can be automated, humans will still be required to ensure patients are compliant and consistent with their medications.</p>
+				<!-- (If you want to learn how to be a Pharma D, click here to read an article by Dhruv Dhall). -->
```
Running into a problem viewing this blog post here (UnexpectedToken). Are you able to see this post? Removing it seems to work.

realsimonwu replied:
It's also preventing rendering for me - didn't catch it before. I've removed the comment in commit 38fc47d9

hirundinidae commented on _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx:
```diff
@@ -7,102 +7,65 @@ authors: ["Prisha Adya"]
+	<div align="center">
+			<img src="/blog/january-2021/chalk-outline.png" alt="Chalk outline of a body" width="500"/>
+	</div>
+	<div align="center">
+			<img src="/blog/january-2021/why-you-should-join-hosa/medical-demo.png" alt="HOSA event medical demo" width="500"/>
+	</div>
```
Did we want to use `<figure>` around all the images? Some of the images on other blog posts use `<figcaption>`. Up to you (keep as `<img>` or add `<figure>` for styling later)

realsimonwu replied:
The images from this particular blogpost don't have any captions or other things that would necessitate `<figure>` - I'll probably keep suing the `<img>` tag here.

hirundinidae replied:
Yep, sounds good. The `<figure>` images are also wrapped with div that centers, don't think there will be any more styling on them.

hirundinidae reviewed (commented):
Looks mostly good, there's one blog post I wasn't able to see without removing the comment. Otherwise, the HTML renders for me :]

realsimonwu committed:
Remove the comment

The HTML comment seems to be preventing this blogpost from rendering when I view it on my own locally running server, as well as when @hirundinidae tries to view it, so I'm removing it.

hirundinidae reviewed (approved):
:+1:
